java/dagger/internal/codegen/DependencyRequest.java

```diff
@@ -370,17 +370,6 @@ private Kind multibindingContributionRequestKind(ContributionBinding multibindin
       return requests.build();
     }
 
-    DependencyRequest forRequiredVariable(VariableElement variableElement) {
```

Comment:
These are unused? Let's keep this PR focused. We can remove them in a different PR.